test/etools-datepicker_test.html

```diff
@@ -20,10 +21,27 @@
 
     <script>
       suite('etools-datepicker', function() {
+        var basic
```

Comment:
I believe `basic` should be `dialog`

Reply:
i think if i name it dialog, it might conflict with the dialog variable that already exists (i think it comes from the element having the id of dialog?):
<paper-dialog id="dialog" class="paper-date-picker-dialog" modal="{{modal}}" with-backdrop="{{withBackdrop}}" on-iron-overlay-closed="dismissDialog">

Reply:
`dialog` might not be a good name in this case, but you should name your variables in a way that it represents the functionality you are creating. This will increase code readability and it's easier for other devs to understand your code. For example, in this case, a good var name would be `etoolsDatepicker`, instead of `basic`.